applications/openshift/api-server/api_server_api_priority_gate_enabled/rule.yml

```diff
@@ -33,9 +33,6 @@ identifiers:
 severity: medium
 
 references:
-  cis@ocp4: 1.2.9
-  nerc-cip: CIP-003-8 R6,CIP-004-6 R3,CIP-007-3 R6.1
-  nist: CM-6,CM-6(1)
   pcidss: Req-2.2
```

Comment:
Should the PCI-DSS control info get removed from the annotations? I didn't find the rule api-server-api-priority-gate-enabled in pci-dss profile.
```
% oc get profile upstream-ocp4-stig -o yaml | grep api-server-insecure-port
- upstream-ocp4-api-server-insecure-port
% oc get profile upstream-ocp4-pci-dss -o yaml | grep api-server-insecure-port
% oc get profile upstream-ocp4-pci-dss-4-0 -o yaml | grep api-server-insecure-port
%
```